bedrock/firefox/templates/firefox/accounts.html

```diff
@@ -56,152 +55,95 @@ <h2 class="c-sub-navigation-title is-summary">
-          <p class="c-product-list-desc">{{ ftl('firefox-accounts-whole-device-protection') }}</p>
-        </a>
-      </li>
-    {% endif %}
-      <li class="c-product-list-item t-product-monitor">
-        <a href="https://monitor.firefox.com?utm_source={{ _utm_source }}&amp;utm_medium=referral&amp;utm_campaign={{ _utm_campaign }}&amp;utm_content=product-list-item&amp;entrypoint={{ _entrypoint }}" data-cta-text="Firefox Monitor" data-cta-type="FxA-Monitor">
-          <h3 class="c-product-list-title">{{ ftl('firefox-accounts-firefox-monitor') }}</h3>
-          <p class="c-product-list-desc">{{ ftl('firefox-accounts-get-a-lookout-for') }}</p>
-        </a>
-      </li>
-    </ul>
-
-    <p class="c-accounts-products-tagline">
-      {% if has_signed_in_copy %}
-        {{ ftl('firefox-accounts-get-it-all-on-every') }}
-      {% else %}
-        {{ ftl('firefox-accounts-and-get-it-all-on') }}
-      {% endif %}
-    </p>
-  </div>
-</section>
-
-
-<section class="c-accounts-value t-value-respect l-media-left">
-  <div class="c-accounts-value-container">
-    <div class="c-accounts-value-media">
-      {{ high_res_img('img/firefox/accounts/trailhead/value-respect.jpg', {'alt': '', 'width': '800', 'height': '600'}) }}
+<section class="mzp-c-split mzp-l-split-reversed">
+  <div class="mzp-c-split-container">
+    <div class="mzp-c-split-media">
+      <div class="mzp-c-emphasis-box l-accounts">
+        <div class="c-accounts-form">
+          {{ fxa_email_form(
+            entrypoint=_entrypoint,
+            form_title=ftl('firefox-accounts-sign-in'),
+            intro_text=ftl('firefox-accounts-enter-your-email-address'),
+            button_class='mzp-c-button mzp-t-primary mzp-t-product')
+          }}
+
+          <p class="fxa-signin">
+            {{ ftl('firefox-accounts-already-have-an-account') }}
+            {{ fxa_button(
+              entrypoint=_entrypoint,
+              button_text=ftl('firefox-accounts-sign-in'),
+              action='signin',
+              is_button_class=False,
+              optional_attributes={'data-cta-text': 'Sign In', 'data-cta-type': 'fxa-sync', 'data-cta-position': 'primary'}
+            ) }}
+          </p>
+        </div>
+
+        <div class="c-manage">
+          <p>You are signed in.</p>
+          {{ fxa_button(
+            entrypoint=_entrypoint,
+            button_text=ftl('firefox-accounts-manage'),
+            action='signin',
+            optional_attributes={'data-cta-text': 'Manage your account', 'data-cta-type': 'fxa-sync', 'data-cta-position': 'primary'}
+          ) }}
+        </div>
+      </div>
     </div>
+    <div class="mzp-c-split-body">
+
+      <div class="accounts-products">
+        <h2>{{ ftl('firefox-accounts-sign-in-to') }}</h3>
+
+        <a href="{{ url('firefox.browsers.index') }}"><h4 class="mzp-c-wordmark mzp-t-wordmark-sm mzp-t-product-firefox">{{ ftl('firefox-accounts-firefox-browser') }}</h4></a>
```

Comment:
Issue (potentially blocking?): it would be nice if these links had some kind of hover state?

Reply:
I debated about adding the links at all. I think having them there with a bad hover state is better than not having them and I don't really want to do any more work :P